concent_api/middleman/constants.py

```diff
@@ -0,0 +1,4 @@
+DEFAULT_INTERNAL_PORT = 8888
+LOCALHOST_IP = "127.0.0.1"
```

Comment:
Shouldn't it be retrieved by code, ie. `socket.gethostbyname(socket.gethostname())` ?

Reply:
no, please see: https://github.com/golemfactory/concent/pull/622#discussion_r201654298